imu:
  frame_id: "guidance"

ultrasonic:
  frame_id: "guidance"

obstacle_distance:
  frame_id: "guidance"

velocity:
  frame_id: "guidance"

motion:
  frame_id: "guidance"
  tf:
    send: true
    frame_id: "world"
    child_frame_id: "guidance"

camera_bottom:
  enable_depth: false
  enable_disparity: false
  enable_left: false
  enable_right: false
  enable_soft_depth: false
  frame_id: "guidance"
  child_frame_id: "cam0"
  sensor_position: {x:  0.0, y:  0.0, z:  0.0} # w.r.t to guidance imu
  orientation: {x: 0.0, y: 0.0, z: 0.0, w: 0.0}

camera_front:
  enable_depth: false
  enable_disparity: false
  enable_left: true
  enable_right: true
  enable_soft_depth: true
  frame_id: "guidance"
  child_frame_id: "cam1"
  sensor_position: {x:  0.0, y:  0.0, z:  0.0} # w.r.t to guidance imu
  orientation: {x: 0.0, y: 0.0, z: 0.0, w: 0.0}

camera_right:
  enable_depth: true
  enable_disparity: false
  enable_left: true
  enable_right: false
  enable_soft_depth: false
  frame_id: "guidance"
  child_frame_id: "cam2"
  sensor_position: {x:  0.0, y:  0.0, z:  0.0} # w.r.t to guidance imu
  orientation: {x: 0.0, y: 0.0, z: 0.0, w: 0.0}

camera_back:
  enable_depth: false
  enable_disparity: false
  enable_left: false
  enable_right: false
  enable_soft_depth: false
  frame_id: "guidance"
  child_frame_id: "cam3"
  sensor_position: {x:  0.0, y:  0.0, z:  0.0} # w.r.t to guidance imu
  orientation: {x: 0.0, y: 0.0, z: 0.0, w: 0.0}

camera_left:
  enable_depth: true
  enable_disparity: false
  enable_left: false
  enable_right: false
  enable_soft_depth: false
  frame_id: "guidance"
  child_frame_id: "cam4"
  sensor_position: {x:  0.0, y:  0.0, z:  0.0} # w.r.t to guidance imu
  orientation: {x: 0.0, y: 0.0, z: 0.0, w: 0.0}

enable_imu: true
enable_utrasonic: true
enable_obstacle_distance: false
enable_velocity: false
enable_motion: false

filterSpeckles: false
maxSpeckleSize: 120
maxDiff: 32
